src/Shared/Data/Database/Items.cs

```diff
@@ -33,6 +33,12 @@ public ItemData Find(string name)
 			return this.Entries.FirstOrDefault(a => a.Value.Name.ToLower() == name).Value;
 		}
 
+		public ItemData FindByClass(string name)
+		{
+			name = name.ToLower();
+			return this.Entries.FirstOrDefault(a => a.Value.ClassName.ToLower() == name).Value;
+		}
```

Comment:
I added this just for utility. We probably will have other instances where we need to search by class name.

Reply:
Absolutely, I wanted to add the same for looking up the default equip.